Apps/WebClient/src/ClientApp/src/components/timeline/encounter.vue

```diff
@@ -0,0 +1,193 @@
+<style lang="scss" scoped>
+@import "@/assets/scss/_variables.scss";
+
+$radius: 15px;
+
+.timelineCard {
+    border-radius: $radius $radius $radius $radius;
+    border-color: $soft_background;
+    border-style: solid;
+    border-width: 2px;
+}
+
+.entryTitle {
+    background-color: $soft_background;
+    color: $primary;
+    padding: 13px 15px;
+    margin-right: -1px;
+    border-radius: 0px $radius 0px 0px;
+}
+
+.icon {
+    background-color: $primary;
+    color: white;
+    text-align: center;
+    padding: 10px 0;
+    border-radius: $radius 0px 0px 0px;
+}
+
+.leftPane {
+    width: 60px;
+    max-width: 60px;
+}
+
+.detailsButton {
+    padding: 0px;
+}
+
+.detailSection {
+    margin-top: 15px;
+}
+
+.commentButton {
+    border-radius: $radius;
+}
+
+.newComment {
+    border-radius: $radius;
+}
+
+.collapsed > .when-opened,
+:not(.collapsed) > .when-closed {
+    display: none;
+}
+</style>
+
+<template>
+    <b-col class="timelineCard">
```

Comment:
Future Improvement Idea: It seems that we can have a common component for timelineCard including many styles and using slots here. But it doesn't have to be in the scope of this task. It may be a new task for refactoring as it will affect other entries as well.